### PR TITLE
Add audit-scanner blog entry

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Fix https://github.com/kubewarden/kubewarden.io/issues/193

## Test

`hugo server -D`

### CODE REVIEW

File: content/blog/2023/09/audit-scanner-feature.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+operations to the resources.
+It then iterates through the tuples, sending to the respective PolicyServers the
+`policy` and `syntethic_admission_request` to evaluate.
+
+Notice how the `synthetic_policy_requests` don't hit the Kubernetes API server;
+there's no need, as no resource is truly being CREATEd, UPDATEd, or DELETEd.
+Instead, the `synthetic_policy_request` is consumed by the PolicyServer. But
+not on its usual `validate/` HTTP endpoint, but in a different `audit/`
+endpoint, exclusive for audit requests. This `audit/` endpoint evaluates the
+policy exactly the same way as the `validate/` one, and is instrumented as
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:40]:
```suggestion
policy in exactly the same way as the `validate/` one, and is instrumented as
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Each Kubernetes Namespace inspected by the Audit Scanner will feature one
+`PolicyReport` object that contains the results of the audit. For cluster-wide
+resources (like Namespace, PersistentVolume, &hellip;) the Audit Scanner
+creates one `ClusterPolicyReport` object per cluster.
+
+There are some resources that the Audit Scanner cannot audit in this first
+implementation. For example, it cannot simulate UPDATE requests, as it doesn't know
+exactly which part of the resource needs to be changed in a meaningful way.
+Also, the DELETE requests will be synthetised in the future. Read the
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:56]:
```suggestion
Also, the DELETE requests will be synthesized in the future. Read the
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+[limitations](https://docs.kubewarden.io/explanations/audit-scanner/limitations)
+doc for an expanded explanation. The full [explanation docs
+page](https://docs.kubewarden.io/explanations/audit-scanner) has a more
+in-depth explanation.
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:60]:
```suggestion
page](https://docs.kubewarden.io/explanations/audit-scanner) has further explanation.
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+The new Audit Scanner brings all this functionality keeping in line with the
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:62]:
```suggestion
The new Audit Scanner brings all this functionality while keeping in line with the
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+modular architecture of Kubewarden. It allows operators to scale it as needed,
+configure its periodicity, monitoring and tracing, and making sure it doesn't
+influence the PolicyServers in a bad way.
+
+#### Fine, but what about a UI?
+
+Reading `(Cluster)PolicyReports` sounds cumbersome?
+
+No fret, the community has created the nice [Policy Reporter UI
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:71]:
```suggestion
Don't worry, the community has created the nice [Policy Reporter UI
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+which is great for the community and the policy engines.
+
+## Do I need to change anything?
+
+No :).
+
+With `1.7.0`, the Audit Scanner is enabled by default. You will notice a new
+`audit-scanner` Cronjob that runs and creates the `(Cluster)PolicyReports`.
+
+If you wish, you can configure its behaviour: have a look at the
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:93]:
```suggestion
If you wish, you can configure its behavior: have a look at the
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+do-not-share-host-paths     default         false      true              monitor   monitor         active   99s
+drop-capabilities           default         true       true              monitor   monitor         active   99s
+no-host-namespace-sharing   default         false      true              monitor   monitor         active   99s
+no-privilege-escalation     default         true       true              monitor   monitor         active   99s
+no-privileged-pod           default         false      true              monitor   monitor         active   99s
+                                                        🠉
+                                                        new spec.backgroundAudit
+```
+
+If you want to skp a policy evaluation, you should set `spec.backgroundAudit`
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:119]:
```suggestion
If you want to skip a policy evaluation, you should set `spec.backgroundAudit`
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+to `false`.
+
+And if you want to provide more metadata, you can 2 new special annotations:
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:122]:
```suggestion
And if you want to provide more metadata, you can use two new special annotations:
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+`io.kubewarden.policy.severity` and `io.kubewarden.policy.category`. Have a
+look at [the
+docs](https://docs.kubewarden.io/explanations/audit-scanner#policies) for their
+values.
+
+## tl:dr;
+
+The new Audit Scanner allows administrators to quickly asses the compliance
+level of their clusters with Kubewarden. It does so by keeping in line with
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:131]:
```suggestion
level of their clusters with Kubewarden. It does so while keeping in line with
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+the existing modular architecture. It allows operators to scale it as needed,
+configure its periodity, its monitoring and tracing, and make sure it doesn't
+influence the PolicyServers in a bad way. We look forward to new policies that
+take advantage of it!
+
+As always, we are curious of what features you would like next and how you are
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:137]:
```suggestion
As always, we are curious as to which features you would like next and how you are
```

**Comment by viccuad** [comment]
Many thanks! applied suggestions.

File: content/blog/2023/09/audit-scanner-feature.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+look at our [Audit Scanner install
+docs](https://docs.kubewarden.io/howtos/audit-scanner) to see how to enable it.
+
+Policy Reporter UI is not the only tool that can consume results from the
+PolicyReports. More and more tools are consuming PolicyReports themselves,
+which is great for the community and the policy engines.
+
+## Do I need to change anything?
+
+No :).
```
Comment (flavio) [inline content/blog/2023/09/audit-scanner-feature.md:87]: Just to ensure proper rendering

```suggestion
No 🙂.
```

**Review by flavio — APPROVED** [review]
Looks great!

**Review by fabriziosestito — APPROVED** [review]

File: content/blog/2023/09/audit-scanner-feature.md (hunk 11 above)
Comment (viccuad) [reply content/blog/2023/09/audit-scanner-feature.md]: https://github.com/kubewarden/kubewarden.io/pull/194/commits/87d198e6578733ee8605fff80cc8904f655c5ec0

**Pull request merged**